README.md

```diff
@@ -48,15 +81,35 @@ How do folks in the community use RFC 4122 UUIDs?
 
 ## FAQ
 
-## History
+**what are the advantages to uuid being a built-in module?**
```

Comment:
s/built-in module/part of the JavaScript standard library/

Reply:
See my comment on the title.